components/LoginComponent/index.tsx

```diff
@@ -20,6 +26,14 @@ interface ILoginComponentProps {
 }
 
 const LoginComponent: FC<ILoginComponentProps> = ({ heading, link, href }) => {
+  const router: NextRouter = useRouter();
+
+  const validationSchema = yup.object().shape({
+    name: yup.string().required(),
```

Comment:
You could extract `yup.string().required()` to a variable, for example, `const requiredField = yup.string().required()`